velox/functions/prestosql/aggregates/ArbitraryAggregate.cpp

```diff
@@ -16,8 +16,8 @@
 
 #include "velox/exec/Aggregate.h"
 #include "velox/expression/FunctionSignature.h"
+#include "velox/functions/lib/SimpleNumericAggregate.h"
 #include "velox/functions/prestosql/aggregates/AggregateNames.h"
```

Comment:
Do we need this include? Can we remove it?

Reply:
I think we cannot remove it, it define functions name.